buddyboss/buddyboss-platform
# Profile Completion widget is not working properly when First or Last …

…name field is disabled

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #895 .

### How to test the changes in this Pull Request:

1. Setup the Profile Completion Widget, and make sure that Details is marked check
2. In BuddyBoss > Settings > Display Name Format, set this to Nickname
3. In BuddyBoss > Settings > Display Name Fields, disable both first and last name fields
4. In BuddyBoss > Profiles > Details Tab, add any new fields

### Proof Screenshots or Video
https://www.loom.com/share/2eeef9367a38437aa43d0ecfdbe7d2d1

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
